create_person.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'classroom'
+
+def create_student(name, age)
+  puts 'Enter the person\'s classroom:'
+  classroom_label = gets.chomp
+  classroom = Classroom.new(classroom_label)
+  puts 'Has parent permission? [y/n]:'
+  parent_permission_input = gets.chomp.downcase
+  parent_permission = parent_permission_input == 'y'
+  Student.new(age, classroom, name, parent_permission: parent_permission)
+end
```

Comment:
- [ ] Great job so far, 👏🏻 However, there is some issue you still have to address before this PR to be approved.
- I'm getting this error 👇🏻 when trying to create a person student 

![image](https://github.com/Ayokunnumi1/school-library/assets/79058364/aad7df02-0a2f-4f72-af6f-6ace0ba4f82c)

- 🤔 And I don't think you need to require the user to Enter the person's classroom:` since it's not on the [Video](https://www.youtube.com/watch?v=vkkgrhD6aXQ)

- [ ] The same for the person teacher after I enter the person's specialization. 👇🏻 

![image](https://github.com/Ayokunnumi1/school-library/assets/79058364/2ace7675-f28c-4843-be85-984413b269ce)